docs/installation.md

```diff
@@ -73,3 +74,7 @@ $ docker-compose -f docker-amundsen.yml up
 9. Verify dummy data has been ingested by viewing in Neo4j by visiting `http://YOUR-DOCKER-HOST-IP:7474/browser/` and run `MATCH (n:Table) RETURN n LIMIT 25` in the query box. You should see two tables -- `hive.test_schema.test_table1` and `dynamo.test_schema.test_table2`.
 10. View UI at `http://YOUR-DOCKER-HOST-IP:5000/table_detail/gold/hive/test_schema/test_table1` or `/table_detail/gold/dynamo/test_schema/test_table2`
 11. View UI at `http://YOUR-DOCKER-HOST-IP:5000` and try to search `test`, it should return some result.
+
+----
+
+> Next: Dive into further [configuration](./configuration.md) | See the [developer guide](./developer_guide.md) to get a good start with hacking on the code.
```

Comment:
This is a helpful feature to guide newcomers, though it creates the responsibility of making sure we direct people to places that make sense. At the moment, since this installation doc serves as a starting point for all Amundsen services it feels fitting to not just direct to the next steps for setting up the frontend service, but all other services as well. 

We don’t know exactly what a developer’s intent will be after they finish the initial setup, so what do you think of adding pointers for each possibility? Wording below is just to give a clear example, feel free to adjust:

> Next Steps: 
> - For application code, dive into further [configuration](./configuration.md)  and see the frontend [developer guide](./developer_guide.md) to get a good start with hacking on the code.
> - To further explore Amunden’s search features, see the search service [developer guide](https://github.com/lyft/amundsensearchlibrary#developer-guide).
> - To further explore Amundsen’s metadata service, see the metadata service [docs] (https://github.com/lyft/amundsenmetadatalibrary#amundsen-metadata-service).
> - To understand how data is ingested, checkout the databuilder [docs](https://github.com/lyft/amundsendatabuilder#amundsen-databuilder).

Reply:
Honestly it feels too heavy for me. Too much focus on inner workings/developer perspective. To little focus on “Quickstart, then next step, next step, etc, 

But it’s a __*very*__ good thing you posted it because it got me to think about overall content structure (not just this file). That’s what I feel we should focus on before this. I think a few *good* “global structure” files are needed. As you said this doc - and even more so the quite lovely architecture.md currently serves that role. This doesn’t necessarily take over on metadata/search documents. They explain HOW/developer reference perspective well - with *local* scope. So these services are still decoupled/reusable in their own right. Great!

Your link to HOW data is ingested in `Databuilder` was really what drove it home for me (or did it make me snap?). *I feel almost embarrassed to say I didn’t think about linking to it after Quickstart* But its a natural next step: “Now I got this thing lit up with demo data, next show me — tutorial style — how to plug in my own data” plus “which data lights up which parts of the UI”. The Databuilder doc is really good, at a deep reference level, it’s even succinct. But at first I was overwhelmed, with too many technical features I failed to know how to apply.

Overall I feel docs need to be written more for the audience on the journey I just posted about on Slack here: https://amundsenworkspace.slack.com/archives/CGFBVT23V/p1557167876290600 (not sure if anyone will ever respond to that, but hey ... at least I could point to it from here now)

For now, I think we should either let this PR sit a bit and see, or simply close it.

Thoughts? Does what I wrote even make any sense or just makes you go🤪?

Reply:
Definitely makes sense. I'm also a fan of reconsidering the structure of our documentation first, which we've been blocked on getting around to but that blocker has now been resolved. In the upcoming weeks, expect to see our repositories and documentation reorganized. 

However once we do that work I'm sure there will still be ways to improve our documentation -- the people who work on code are usually not a good litmus test for how good the documentation is. 😅 

What if we leave this PR open to sit, continue to take input as we restructure our documentation in the upcoming weeks, and then circle back to see how the community thinks we can still improve?

Reply:
Sounds good to me!

Looking forward to see what's cooking...